packages/block-library/src/image/styles.native.scss

```diff
@@ -16,6 +20,10 @@
 	align-items: center;
 }
 
+.caption-text {
```

Comment:
Nit: for simplicity in accessing it, and to match the rest of the class definitions in the same file, what do you think about renaming it to `.captionText`?

Reply:
Yes, `captionText` was my 1st choice, but gets an error in JS lint if I rename it:
```
[0] gutenberg-mobile/gutenberg/packages/block-library/src/image/edit.native.js
[0]   294:57  error  ["captionText"] is better written in dot notation  dot-notation
```

Reply:
Aha, apparently the rules weren't in place in the past and the other class definitions had passed. Oh well, `caption-text` it is then :)

Reply:
I think that message is not about the dash, but it's telling you to replace `styles['captionText']` with `styles.captionText`